Python/test.py

```diff
@@ -0,0 +1,35 @@
+import hashlib
+import base64
+import json
+
+test_payload = {
+    "merchantId": "DemoMerchant",
+    "transactionId": "TX123456789",
+    "merchantOrderId": "M123456789",
+    "amount": 100,
+    "instrumentType": "MOBILE",
+    "instrumentReference": "9xxxxxxxxxx",
+    "message": "collect for XXX order",
+    "email": "amitxxx75@gmail.com",
+    "expiresIn": 180,
+    "shortName": "DemoCustomer",
+    "subMerchant": "DemoMerchant",
+    "storeId": "store1",
+    "terminalId": "terminal1"
+}
+request = json.dumps(test_payload, separators=(',', ':')) # compact encoding
+base64_payload = base64.urlsafe_b64encode(
+    bytes(request, "utf-8")).decode("utf-8")
+
+VERIFICATION_HASH = base64_payload + "/v3/charge" + 'key1'
```

Comment:
@paramaggarwal Can we use the CHARGE_ENDPOINT constant here?

Reply:
This is just the test file. Here everything is hardcoded as the final expected has is also hardcoded. See the `key1` is also hardcoded. This is only meant to test hashing/signing logic.